src/operations/working-groups.md

```diff
@@ -0,0 +1,59 @@
+# Working Groups
+We use [Teams](https://github.com/orgs/tomeshnet/teams/toronto-community-network/teams) to manage membership of working groups and [Projects](https://github.com/tomeshnet/toronto-community-network/projects/1) to track task status on a public project board.
```

Comment:
```suggestion
We use [GitHub Teams](https://github.com/orgs/tomeshnet/teams/toronto-community-network/teams) to manage membership of working groups and [GitHub Projects](https://github.com/tomeshnet/toronto-community-network/projects/1) to track task status on a public project board.
```

`Teams` can be confused with Microsoft Teams, a popular collaboration/video conferencing platform from Microsoft.